### PR TITLE
chore: remove the bff-sdk directory & add the command gql-sdk-upgrade to the Makefile

### PR DESCRIPTION
**Pull request by dayuy**
#### What type of PR is this?
/kind cleanup
/kind flake

#### What this PR does / why we need it
remove the bff-sdk directory & add the command gql-sdk-upgrade to the Makefile. automatically upgrade the SDK version.

#### Which issue(s) this PR fixes

Fixes #

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -254,6 +254,10 @@ gql-gen:
 build-graphql-server: gql-gen
 	@CGO_ENABLED=0 GOOS=linux go build -o bin/graphql-server graphql-server/go-server/main.go
 
+# sdk for graphql-server api
+gql-sdk-upgrade:
+	docker run -v $(shell pwd)/graphql-server/go-server/graph/schema:/schema 172.22.96.34/dev-branch/arcadia-bff-sdk:main
```
Comment (0xff-dev) [inline Makefile:259]: Perhaps we should push the image  `172.22.96.34/dev-branch/arcadia-bff-sdk:main` to `dockerhub`. @bjwswang @Abirdcfly

Comment (Abirdcfly) [reply Makefile:259]: Taking it a step further, should we make the `make gql-sdk-upgrade` command auto-triggered to automatically run in github action as soon as some files change?

Comment (bjwswang) [reply Makefile:259]: 1st. Never tag/use image with a internal image registry. Cause we are opensourced project!
2nd. yes. Make this a github action and triggered automatically. The generated sdk should be published to a publish npm registry.

Comment (dayuy) [reply Makefile]: 1. Using the image `kubebb/gql-sdk-generator:main`
2. Put the Github action on my to do list, I  need to learn how to write the workflow.

**Comment by dayuy** [comment]
Usage:

If you have made changes to any  *.gql files, you need to follow the steps below before pushing the commit.

1. `make gql-sdk-generator`
2.  check whether the SDK has been published successfully

**Review by Abirdcfly — APPROVED** [review]

**Review by bjwswang — APPROVED** [review]

**Pull request merged**